systemtest/src/test/java/io/strimzi/systemtest/security/NetworkPoliciesST.java

```diff
@@ -0,0 +1,334 @@
+            .endSpec()
+            .done();
+
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3).done();
+
+        checkNetworkPoliciesInNamespace(NAMESPACE);
+
+        changeKafkaConfigurationAndCheckObservedGeneration(NAMESPACE);
+    }
+
+    @Test
+    void testNPWhenOperatorIsInDifferentNamespaceThanOperand() {
+        String secondNamespace = "second-" + NAMESPACE;
+
+        Map<String, String> labels = new HashMap<>();
+        labels.put("my-label", "my-value");
+
+        EnvVar operatorLabelsEnv = new EnvVarBuilder()
+            .withName("STRIMZI_OPERATOR_NAMESPACE_LABELS")
+            .withValue(labels.toString().replaceAll("\\{|}", ""))
+            .build();
+
+        cluster.createNamespace(secondNamespace);
+
+        prepareEnvForOperator(NAMESPACE, Arrays.asList(NAMESPACE, secondNamespace));
+
+        // Apply role bindings in CO namespace
+        applyRoleBindings(NAMESPACE);
+
+        // Create ClusterRoleBindings that grant cluster-wide access to all OpenShift projects
+        List<ClusterRoleBinding> clusterRoleBindingList = KubernetesResource.clusterRoleBindingsForAllNamespaces(NAMESPACE);
+        clusterRoleBindingList.forEach(clusterRoleBinding ->
+            KubernetesResource.clusterRoleBinding(clusterRoleBinding, NAMESPACE));
+        // 060-Deployment
+        BundleResource.clusterOperator("*", Constants.CO_OPERATION_TIMEOUT_DEFAULT)
+            .editOrNewSpec()
+                .editOrNewTemplate()
+                    .editOrNewSpec()
+                        .editContainer(0)
+                            .addToEnv(operatorLabelsEnv)
+                        .endContainer()
+                    .endSpec()
+                .endTemplate()
+            .endSpec()
+            .done();
+
+        kubeClient().getClient().namespaces().withName(NAMESPACE).edit()
+            .editMetadata()
+                .addToLabels(labels)
+            .endMetadata()
+            .done();
+
+        cluster.setNamespace(secondNamespace);
+
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3)
+            .editMetadata()
+                .addToLabels(labels)
+            .endMetadata()
+            .done();
+
+        checkNetworkPoliciesInNamespace(secondNamespace);
+
+        changeKafkaConfigurationAndCheckObservedGeneration(secondNamespace);
+    }
+
+    void checkNetworkPoliciesInNamespace(String namespace) {
+        List<NetworkPolicy> networkPolicyList = new ArrayList<>(kubeClient().getClient().network().networkPolicies().inNamespace(namespace).list().getItems());
+
+        String networkPolicyPrefix = CLUSTER_NAME + "-network-policy";
+
+        assertNotNull(networkPolicyList.stream().filter(networkPolicy ->  networkPolicy.getMetadata().getName().contains(networkPolicyPrefix + "-kafka")).findFirst());
+        assertNotNull(networkPolicyList.stream().filter(networkPolicy ->  networkPolicy.getMetadata().getName().contains(networkPolicyPrefix + "-zookeeper")).findFirst());
```

Comment:
Just a question: shouldn't we check some specific value in NP? Just asking.

Reply:
Maybe I'm wrong but I don't think that we need to check some values in NP.. because you only need to know that NP for Kafka and Zookeeper are created, the values inside are tested in some UT maybe..